traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -72,7 +72,9 @@ class AvailabilityViewController: UIViewController {
         }
 
         guard let availability = selectedAvailability else {
-            availabilityError = BookingError.noDate
+            if availabilityError == nil {
+                availabilityError = BookingError.noDate
+            }
```

Comment:
this is now unnecessary

it will never happen, user will only enter this screen if they have selected a date

Reply:
This screen is for selecting a date.